test/credentials/cognitiveServicesCredentialsTests.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import * as assert from "assert";
+import { WebResource } from "ms-rest-js";
+import { CognitiveServicesCredentials } from "../../lib/credentials/cognitiveServicesCredentials";
+import * as msAssert from "../msAssert";
+
+describe("CognitiveServicesCredentials", () => {
```

Comment:
Are these tests migrating in from ms-rest-azure?

Reply:
Nope. I just wrote these to get test coverage.